pkg/controller/plan/adapter/ova/builder.go

```diff
@@ -349,19 +353,25 @@ func (r *Builder) mapCPU(vm *model.VM, object *cnv.VirtualMachineSpec) {
 }
 
 func (r *Builder) mapFirmware(vm *model.VM, object *cnv.VirtualMachineSpec) {
+	migrationAnnotations := r.Migration.GetAnnotations()
+	virtV2VFirmware := migrationAnnotations[vm.ID]
+
 	firmware := &cnv.Firmware{
 		Serial: vm.UUID,
 	}
-	switch vm.Firmware {
+
+	secureBootEnabled := false
```

Comment:
we can use the ptr package to avoid declaring secureBootEnabled

Reply:
//TODO